Sources/Segment/Utilities/HTTPClient.swift

```diff
@@ -169,11 +169,4 @@ extension HTTPClient {
 
         return request
     }
-
-    internal static func configuredSession(for writeKey: String) -> URLSession {
```

Comment:
It turns out, the write key is never used in the configuration of the session so we don't need this function I don't believe. Which means we can wrap up the creation of the session in the `DefaultHTTPSession` enum.